components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Models/BaseMonitoredItemModel.cs

```diff
@@ -69,23 +70,51 @@ public class MonitoredItemModel {
         public string TriggerId { get; set; }
 
         /// <summary>
-        /// Data change filter
+        /// Clones this object
         /// </summary>
-        public DataChangeFilterModel DataChangeFilter { get; set; }
+        /// <returns></returns>
+        public abstract BaseMonitoredItemModel Clone();
 
         /// <summary>
-        /// Event filter
+        /// Compare items
         /// </summary>
-        public EventFilterModel EventFilter { get; set; }
-
-        /// <summary>
-        /// Aggregate filter
-        /// </summary>
-        public AggregateFilterModel AggregateFilter { get; set; }
-
-        /// <summary>
-        /// heartbeat interval not present if zero
-        /// </summary>
-        public TimeSpan? HeartbeatInterval { get; set; }
+        /// <param name="other"></param>
+        /// <returns></returns>
```

Comment:
Please remove or will out

Reply:
I don't understand what you mean here

Reply:
Now I understand and I have fixed it. Did a quick search and there are 2430 other occurrences of this in the code :-)